src/docker-compose.yaml

```diff
@@ -0,0 +1,107 @@
+            # HTTPS
+            - "traefik.http.routers.example-app-https-router.rule=Host(`example-app.blumilk.localhost`)"
+            - "traefik.http.routers.example-app-https-router.entrypoints=websecure"
+            - "traefik.http.routers.example-app-https-router.tls=true"
+        container_name: example-app-app-dev
+        working_dir: /application
+        volumes:
+            - ./environment/dev/app/nginx.conf:/etc/nginx/nginx.conf:ro
+            - ./environment/dev/app/php.ini:/usr/local/etc/php/conf.d/zzz-overrides.ini:ro
+            - ./environment/dev/app/php-fpm.conf:/usr/local/etc/php-fpm.d/zzz-overrides.conf:ro
+            - ./environment/dev/app/supervisord.conf:/etc/supervisor/custom-supervisord.conf:ro
+            - .:/application
+        ports:
+            - ${DOCKER_APP_HOST_PORT:-3851}:80
+        networks:
+            - example-app-dev
+            - traefik-proxy-blumilk-local
+        restart: unless-stopped
+        depends_on:
+            database:
+                condition: service_healthy
+
+    database:
+        image: postgres:15.2-alpine3.17
+        container_name: example-app-db-dev
+        environment:
+            - POSTGRES_USER=${DB_USERNAME}
+            - POSTGRES_PASSWORD=${DB_PASSWORD}
+            - POSTGRES_DB=${DB_DATABASE}
+            - PGDATA=/var/lib/postgresql/data
+        healthcheck:
+            test: [ "CMD-SHELL", "pg_isready --dbname ${DB_DATABASE} --username ${DB_USERNAME}"]
+            interval: 3s
+            timeout: 3s
+            retries: 5
+        ports:
+            - ${DOCKER_DATABASE_HOST_PORT:-3853}:5432
+        volumes:
+            - example-app-postgres-data:/var/lib/postgresql/data
+        networks:
+            - example-app-dev
+        restart: unless-stopped
+
+    mailpit:
+      image: axllent/mailpit:v1.8.0
+      container_name: example-app-mailpit-dev
+      labels:
+        - "traefik.enable=true"
+        - "traefik.blumilk.environment=true"
+        # HTTP
+        - "traefik.http.routers.example-app-mailpit-http-router.rule=Host(`example-app-mailpit.blumilk.localhost`)"
+        - "traefik.http.routers.example-app-mailpit-http-router.entrypoints=web"
+        # HTTP to HTTPS redirect
+        #      - "traefik.http.routers.erasmus-mailpit-http-router.middlewares=https-redirect@file"
+        # HTTPS
+        - "traefik.http.routers.example-app-mailpit-https-router.rule=Host(`example-app-mailpit.blumilk.localhost`)"
+        - "traefik.http.routers.example-app-mailpit-https-router.entrypoints=websecure"
+        - "traefik.http.routers.example-app-mailpit-https-router.tls=true"
+        # LOADBALANCER MAILPIT PORT
+        - "traefik.http.services.example-app-mailpit.loadbalancer.server.port=8025"
+      networks:
+        - example-app-dev
+        - traefik-proxy-blumilk-local
+      ports:
+        - ${DOCKER_MAILPIT_DASHBOARD_HOST_PORT:-63854}:8025
+      restart: unless-stopped
+
+    redis:
+        image: redis:7.0.11-alpine3.17
```

Comment:
`7.2.3-alpine3.18` is the latest at this moment.

Reply:
updated